specs-go/config.go

```diff
@@ -300,7 +300,7 @@ type LinuxCPU struct {
 // LinuxPids for Linux cgroup 'pids' resource management (Linux 4.3)
 type LinuxPids struct {
 	// Maximum number of PIDs. Default is "no limit".
-	Limit *int64 `json:"limit,omitempty"`
+	Limit int64 `json:"limit"`
```

Comment:
@crosbymichael, `nil` is not allowed because [`limit` is required][1].

@stevvooe, I expect 0 should be illegal.  Can you file a separate PR to require strictly positive value?

[1]: https://github.com/opencontainers/runtime-spec/blob/e91c55e9a13d36200a1bb6a233ee9583761d1764/config-linux.md#pids